.github/pull_request_template.md

```diff
@@ -10,3 +10,4 @@
 - [ ] Closes #xxx
 - [ ] Tests added/passed
 - [ ] Documentation updated
+- [ ] Telemetry compliance tasks completed for any telemetry events added/modified following the guidance in [aka.ms/devhome-telemetry](aka.ms/devhome-telemetry)
```

Comment:
This link doesn't work -- without the https:// it thinks it's a relative github url